docker-compose/.env

```diff
@@ -1,11 +1,14 @@
 TITLE="MariaDB CI"
 TITLE_URL=https://github.com/MariaDB/server
 BUILDMASTER_URL=https://buildbot.mariadb.org/
+CR_HOST_WG_ADDR="hz-bbw5=100.64.100.20"
 MQ_ROUTER_URL=ws://localhost:8085/ws
 MASTER_PACKAGES_DIR="/mnt/autofs/master_packages"
 GALERA_PACKAGES_DIR="/mnt/autofs/galera_packages"
 ARTIFACTS_URL="https://ci.mariadb.org"
 NGINX_ARTIFACTS_VHOST="ci.mariadb.org"
+NGINX_ARTIFACTS_SSL_PATH="ci.mariadb.org"
```

Comment:
This should be symmetric between DEV/PROD

Reply:
Then we need to reconfigure a bit the DEV environment so we generate one certificate per service. 
RN we have:
![image](https://github.com/user-attachments/assets/c818b69b-1696-423f-87b3-6308d32a7dbb)